src/utils/JobQueueMock.ts

```diff
@@ -4,6 +4,24 @@ import { JobStore } from '../models/JobStore';
 export class JobStoreMock implements JobStore {
     jobs: RawJob[] = [];
     constructor() {}
+
+    sortJobs(jobs: RawJob[]) {
```

Comment:
Can you make this function private and use the returned sorted arrays instead of using the inplace sort (and return the sorted array at the end)? I think this would improve the readability.

Reply:
Sure I can do, but just one thing: the `jobs` array passed as a parameter will still be sorted in place so it maybe is kind of redundant to return it extra. But I can do it for readability!

Reply:
done, please review again @SimonErm

Reply:
ah sorry, not quite yet...

Reply:
now 😄

Reply:
I was confuse because the commit with the changes was named "format code"😄 Looks good now :-)